spec/controllers/animals_controller_spec.rb

```diff
@@ -57,6 +63,28 @@
           expect(new_animal).to_not be_nil
         end
       end
+
+      context 'exitoso por un usuario con permiso de super usuario' do
+
+        before(:each) do
+          request.headers['X-USER-TOKEN'] = user3.authentication_token
+        end
+
+        it 'devuelve código 201 create' do
+          post :create, animal: params, format: 'json'
```

Comment:
La llamada `post :create, animal: params, format: 'json'` aparece en los 3 it de este context. Se podría poner en el bloque before(:each)? (La tercera es muy parecedia, si la modificas un poco te queda igual)